src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -113,165 +110,173 @@ const MapView: React.FC<MapViewProps> = ({
     if (mapElement && treePopupElement) {
       const markersArray: google.maps.Marker[] = [];
 
-      LOADER.load().then(() => {
-        map = new google.maps.Map(mapElement, {
-          center: BOSTON,
-          zoom: 12,
-          fullscreenControl: false,
-          mapTypeControl: false,
-          // mapId: mapId,
-          restriction: {
-            latLngBounds: BOSTON_BOUNDS,
-            strictBounds: false,
-          },
-        });
+      LOADER.load()
+        .then(() => {
+          map = new google.maps.Map(mapElement, {
+            center: BOSTON,
+            zoom: 12,
+            fullscreenControl: false,
+            mapTypeControl: false,
+            // mapId: mapId,
```

Comment:
❓  Should this stay commented out for any reason or can we delete this?

Reply:
It doesn't seem like there's a reason to me, will look into it/take it out in the map cleanup ticket